src/main/java/Entities/Gradable.java

```diff
@@ -0,0 +1,25 @@
+package Entities;
+
+public interface Gradable {
+    double weightage = 0;
+    double gradeReceived = -1; // the grade the user receives, -1 if not yet received
+    double gradeGoal = 100; // the user's grade goal
```

Comment:
Gradable tasks no longer need to have a gradeGoal attribute. We changed the user story so that grade goals only happen on a course basis and not a task basis. You can delete this attribute and the corresponding set method.

Reply:
Fixed